### DISCUSSION TITLE
[Feature Request] Org mode editor

### DISCUSSION THREAD

**Discussion by kusuriya** [standardnotes/forum#2491] [Suggestions]
It would be nice to be able to take notes using org mode syntax.

**Comment by jrabensc** [comment]
https://github.com/ryanpcmcquen/standardnotes_org_mode_editor

**Comment by mfaerevaag** [comment]
For my use, it would be most usable if it is simply added as a "language" in the vim editor

**Comment by JaspalSuri** [comment]
I'll take note of both of your requests. Thanks for the feedback!
